### ISSUE TITLE
[QUESTION]  Predicting a Single Data Point from a Complete Time Series

### ISSUE THREAD

**Issue by k28998989** [unit8co/darts#2703]
Can I use a complete time series to predict a single data point within the same time series?

I want to predict a specific row in series, but I'm not sure where to add the prediction step. Below is my code:

series = TimeSeries.from_dataframe(raw_data)  # This dataset has five columns
model = RNNModel(
    input_chunk_length=10, model='LSTM', hidden_dim=25, n_rnn_layers=3, 
    dropout=0.0, training_length=20, batch_size=16, force_reset=True, n_epochs=200
)

model.fit(series=train_target)

**Comment by madtoinou** [comment]
Hi @k28998989,

I would recommend reading the Darts [User Guide](https://unit8co.github.io/darts/userguide/torch_forecasting_models.html) and maybe also run one of the example notebook, such as [Regression Model example](https://unit8co.github.io/darts/examples/20-RegressionModel-examples.html) or [RNN Model](https://unit8co.github.io/darts/examples/04-RNN-examples.html), they contain all the information necessary to solve your problem.

As guidance, your problem is **univariate** (you want to forecast only one column), the other columns can probably be considered as **past covariates** (or can simply be removed, if they are not informative for the target column). Also, as a side note, Darts always assumes that you want the predict the time-steps following the end of the training time-series.
